src/utils/index.ts

```diff
@@ -4,7 +4,14 @@ export function formatPrice(amount: any) {
 }
 
 export function calculateTotal(cart) {
-  return cart?.reduce((accumulator, item) => parseFloat(item.price) * item.quantity + accumulator, 0);
+  try {
+    return cart?.reduce(
+      (accumulator, item) => parseFloat(item.price) * (1 - item.discount / 100) * item.quantity + accumulator,
```

Comment:
what if the item does not have a discount

Reply:
hm, I think discount is 0 by default. I'll add || 0 so just incase